multisense_ros/src/camera_utilities.cpp

```diff
@@ -349,7 +349,7 @@ Eigen::Vector3f StereoCalibrationManager::reproject(size_t u,
                                                     size_t v,
                                                     double d,
                                                     const sensor_msgs::CameraInfo &left_camera_info,
-                                                    const sensor_msgs::CameraInfo &right_camera_info) const
+                                                    const sensor_msgs::CameraInfo &right_camera_info)
```

Comment:
Shouldn't this still be const?

Reply:
good catch

Reply:
Nevermind, this is now a static function so it can't be const. The goal is that the function can be used without having to construct an instance of the class